packages/wearable_rotary/tizen/src/wearable_rotary_plugin.cc

```diff
@@ -0,0 +1,85 @@
+#include "wearable_rotary_plugin.h"
+
+#include <efl-extension/wearable/circle/efl_extension_rotary.h>
+#include <flutter/encodable_value.h>
+#include <flutter/event_channel.h>
+#include <flutter/event_sink.h>
+#include <flutter/event_stream_handler_functions.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_method_codec.h>
+
+#include <memory>
+
+#include "log.h"
+
+#define ROTARY_CHANNEL_NAME "flutter.wearable_rotary.channel"
+
+class WearableRotaryPlugin : public flutter::Plugin {
+ public:
+  static void RegisterWithRegistrar(flutter::PluginRegistrar *registrar) {
+    LOG_DEBUG("RegisterWithRegistrar for WearableRotaryPlugin");
+    auto plugin = std::make_unique<WearableRotaryPlugin>();
+    plugin->setupEventChannels(registrar);
+    registrar->AddPlugin(std::move(plugin));
+  }
+
+  WearableRotaryPlugin() {}
+
+  virtual ~WearableRotaryPlugin() {}
+
+ private:
+  void setupEventChannels(flutter::PluginRegistrar *registrar) {
+    channel_ = std::make_unique<flutter::EventChannel<flutter::EncodableValue>>(
+        registrar->messenger(), ROTARY_CHANNEL_NAME,
+        &flutter::StandardMethodCodec::GetInstance());
+    auto wearable_rotary_channel_handler =
+        std::make_unique<flutter::StreamHandlerFunctions<>>(
+            [this](const flutter::EncodableValue *arguments,
+                   std::unique_ptr<flutter::EventSink<>> &&events)
+                -> std::unique_ptr<flutter::StreamHandlerError<>> {
+              LOG_DEBUG("OnListen");
+              Eina_Bool ret =
+                  eext_rotary_event_handler_add(RotaryEventCallBack, this);
+              if (ret == EINA_FALSE) {
+                LOG_ERROR("[WearableRotaryPlugin] failed_to_add_callback");
```

Comment:
Could you remove _ in all log messages?

Reply:
Also note that we don't even need this kind of error log and simply remove if it doesn't carry any additional information.

Reply:
@swift-kim Could you elaborate on that?

Reply:
The error reason is already available through the PlatformException instance so you don't typically have to repeat the same thing in the plugin log. Our plugins tend to abuse logs too much. Also you don't need the prefix [WearableRotaryPlugin] since the plugin's log tag is already set to it.